examples/data-sources/utils_spacelift_stack_config/stacks/uw2-dev.yaml

```diff
@@ -51,6 +51,8 @@ components:
           autodeploy: true
           branch: "test"
           triggers: []
+          depends-on:
```

Comment:
`depends_on` > `depends-on` to stay consistent with other keys in YAML Stack schema?